cmd/crossplane/main.go

```diff
@@ -20,11 +20,12 @@ import (
 	"log"
 	"time"
 
-	"github.com/crossplaneio/crossplane/pkg/apis"
-	"github.com/crossplaneio/crossplane/pkg/controller"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	"github.com/crossplaneio/crossplane/pkg/apis"
```

Comment:
to keep all the imports sorted, will we have to continually run the commands mentioned in the commit message? or has that been automated somehow already?

```
$ goimports -w -local "github.com/crossplaneio/crossplane/" $(find cmd/ -name *.go|xargs)
$ goimports -w -local "github.com/crossplaneio/crossplane/" $(find pkg/ -name *.go|xargs)
```

Reply:
`goimports` is a drop-in replacement for `gofmt`, so assuming you have an editor hook that runs `gofmt` for you the trick will be to replace it with `gofmt -local "github.com/crossplaneio/crossplane/". I personally use VS Code, which has an option to run `goimports` rather than `gofmt`.

Reply:
We could perhaps add a `make fmt` command to automate this for folks who don't want to setup editor integration.

Reply:
should that setup step be added to our contributor guide? we added something similar in Rook recently with https://github.com/rook/rook/blob/master/Documentation/development-flow.md#development-settings